### ISSUE TITLE
Accessibility issues for Facebook universal app

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4007]
**Reported by supanut2000 on 2014-03-21 12:24**
NVDA often doesn't read Facebook app's interface right at the first time the app is launched, and it also stops reading the interface as well if the app was minimized or I open another metro app, for example the Windows Store app.

**Comment by leonardder** [comment]
It is likely that Facebook is devoting all their work to the Windows 10 app. I'm happy to change the title of this issue to cover the Windows 10 app for facebook, if there are similar problems in that app.

CC @JosephSL

**Comment by k-kolev1985** [comment]
@leonardder Yes, the Facebook team is concentrated on the app for Windows 10. But its current state in regards to accessibility is not very good. The log in page is not accessible at all, it seams. If you manage to log in, you can read controls with TAB. But that is about it, based on my observations. Hope it will improve soon, but who knows...

**Comment by josephsl** [comment]
Hi, yes, please do change the title. Also, it might be time to send a stronger love letter to Facebook regarding accessibility problems in their universal app. Thanks.

**Comment by jcsteh** [comment]
Ideally, these issues should be reported to (and fixed by) Facebook. It may not even be possible to work around these in NVDA. I'll set to p4 in case someone wants to look into this further, though.

**Comment by Adriani90** [comment]
@supanut2000 could you test the app and give an update regarding its accessibility? Is this still as bad as before?

**Comment by Adriani90** [comment]
I have tested in NVDA 2019.1.1 with the last version of the facebook app from Windows store, the app is still very inaccessible. Edit fields are not in the tab order, you cannot jump to them with objectnavigation and you cannot move into the text etc.
cc: @josephsl I supose the priority for improving Windows 10 aps in general is very low. Right? It seems many companies just create some apps to test the Windows store but the users of Windows are not really using lots of store apps. My impression is that people using store apps are either on Android or on Apple. So I think in the future there will not be much improvement to accessibility of Windows store apps.

**Comment by Adriani90** [comment]
The Facebook App from Microsoft App store is now fully accessible, I cannot find any really significant problem with it using NVDA last alpha. I am closing as works for me. If you have individual issues with the app, please open new issues and describe them according to the issue template.
